packages/config/src/cached_config.js

```diff
@@ -27,6 +24,6 @@ const parseCachedConfig = async function (cachedConfig, cachedConfigPath) {
   }
 
   if (cachedConfigPath !== undefined) {
-    return JSON.parse(await pReadFile(cachedConfigPath, 'utf8'))
+    return JSON.parse(await fs.readFile(cachedConfigPath))
```

Comment:
Nice removal of string encoding passed to `JSON.parse()`. Thanks!

Reply:
I think this was just autofixed now :)

Reply:
Great job from `eslint-plugin-unicorn` folks on that one :)